draft-ietf-httpbis-http2bis.xml

```diff
@@ -1441,24 +1451,33 @@
           a header section or a trailer section. HEADERS frames can be sent on a stream
           in the "idle", "reserved (local)", "open", or "half-closed (remote)" state.
         </t>
-        <figure anchor="HEADERSFramePayload">
-          <name>HEADERS Frame Payload</name>
+        <figure anchor="HEADERSFrameFormat">
+          <name>HEADERS Frame Format</name>
           <artwork type="inline"><![CDATA[
- +---------------+
- |Pad Length? (8)|
- +-+-------------+-----------------------------------------------+
- |E|                 Stream Dependency? (31)                     |
- +-+-------------+-----------------------------------------------+
- |  Weight? (8)  |
- +-+-------------+-----------------------------------------------+
- |                    Field Block Fragment (*)                 ...
- +---------------------------------------------------------------+
- |                           Padding (*)                       ...
- +---------------------------------------------------------------+
+Headers Frame {
+  Length (24),
+  Type (8),
+  END_STREAM Flag (1),
+  Unused Flag (1),
+  END_HEADERS Flag (1),
+  PADDED Flag (1),
+  Unused Flag (1),
+  PRIORITY Flag (1),
+  Unused Flags (2),
+  R (1),
+  Stream Identifier (31),
+  [Pad Length (8)],
+  E (1),
```

Comment:
Similarly for `E` here, can we use the word `Exclusive`?

Reply:
We already have an Exclusive Flag; having an Exclusive Field is going to confuse people. Can we come up with another name that distinguishes them a bit more?

Reply:
Oh wait, no we don't -- that's just loose use of the term 'flag' in 7540. Hold on a sec...